gestalt-entity-system/src/main/java/org/terasology/gestalt/entitysystem/component/EmptyComponent.java

```diff
@@ -0,0 +1,16 @@
+package org.terasology.gestalt.entitysystem.component;
+
+/**
+ * An abstract component to be extended if a component that has no information to copy, like components that are flags
+ * for properties.
+ * <p>
+ * This should only be used for components that contain no information. If this is used for a component that does
+ * contain data, that data could be lost.
```

Comment:
It is a bit worse than data could be lost. The uncopied information would never get saved into the component store, so the component will not work as expected at all.